editor/src/core/webpack-loaders/default-loader.ts

```diff
@@ -0,0 +1,16 @@
+import { LoadModule, loadModuleResult, MatchFile, ModuleLoader } from './loader-types'
+
+const matchFile: MatchFile = (filename: string) => {
+  return ['.js', '.jsx', '.ts', '.tsx', '.d.ts', '.json'].some((extension) =>
+    filename.endsWith(extension),
+  )
+}
+
+const loadModule: LoadModule = (filename: string, contents: string) => {
+  return loadModuleResult(filename, contents)
```

Comment:
Add a FIXME me here to investigate whether we should run the Babel transform for converting to Common JS